clearml-server-cloud-ready/templates/network-policy-agentservices.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "clearml.fullname" . }}-agentservices
+  labels:
+    {{- include "clearml.labels" . | nindent 4 }}
+spec:
+  egress:
+  - ports:
+    - port: 8008
+      protocol: TCP
+    to:
+    - podSelector:
```

Comment:
Hi Jake,

I didn't notice connections from agent-services to fileserver on my installation. Maybe because I just deployed it and it is actively used yet. We use kube-iptables-tailer to see dropped packets between k8s pods and it didn't show any dropped packets between agent-services and fileserver, that's why I didn't add a network policy for this.

So, the following connection is required, right?
From: agent-services
To: fileserver:8081

If, yes, I will create one more network policy for the connection above.

As for access to various other services - I have no a complete list of all the required connections. If they (connections) arise - I'd add network policies for all of them.

Reply:
Well, it's:
- apiserver -> elastic
- apiserver -> mongodb
- apiserver -> redis
- webserver -> apiserver
- webserver -> fileserver
- agent-services -> apiserver
- agent-services -> fileserver

And in the future, possibly:
- webserver -> agent-services (future)
- fileserver -> apiserver
- fileserver -> redis